src/containers/Graph/forceGraphConfig.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { nodeCanvasObject } from './nodeCanvasObject';
+import ToolTip from './toolTip';
+
+const layouts = {
+  'force-directed': undefined,
+  tree: 'td',
+  radial: 'radialout',
+};
+
+export const forceGraphConfig = ({ ...rest }) => ({
+  onNodeClick: rest.handleNodeClick,
+  // onLinkClick: rest.handleNodeClick,
+  width: rest.width,
+  height: rest.height,
+  nodeLabel: (node) => `<div>${ToolTip({ data: node })}</div>`,
+
+  linkLabel: (link) => `<div>${ToolTip({ data: link })}</div>`,
+  // onNodeHover: (node) => {
+  //   // Handle node hover event
+  //   console.log('Hovered node:', node);
+  //   rest.setTooltipNode(node);
+  // },
+  // onNodeBlur: () => {
+  //   // Handle node blur event
+  //   rest.setTooltipNode(null);
+  // },
+  // linkDistance: 10000,
+  // nodeSpacing: 8000,
+  // forceX: 7.1,
+  // forceY: 8.1,
+  dagMode: layouts['force-directed'],
+  linkLabelOffset: 10,
+  linkDirectionalArrowLength: 2,
+  linkDirectionalArrowRelPos: 10,
+  nodeRelSize: 10,
+  // linkCurvature: 1.5,
+  linkWidth: 0.4,
+  backgroundColor: '#FFFFFF',
+  nodeRenderer: rest.customNodeRenderer,
+  onNodeDragEnd: (node) => {
+    node.fx = node.x;
+    node.fy = node.y;
+    node.fz = node.z;
+  },
+  warmUpTicks: 100,
+  cooldownTicks: 100,
+  onNodeCreated: (node) => {},
+  // d3Force: 'charge',
+  // d3ForceParams: { distanceMax: 100, forceStrength: -500 },
+  linkColor: () => '#b5b3b3',
+  // zoomToFit: 200,
+  nodeVal: (node) => (node.label === 'Entity' ? 15 * 4.25 : 6),
```

Comment:
`nodeVal` is a rendering engine property that needs to match actual size of the node on the screen (this property defines perimeter of the node circle for collision etc).
```suggestion
  nodeVal: (node) => (node.labels.includes('Entity') ? 8 : 0.25),
```